ReadMe.md

```diff
@@ -31,6 +31,13 @@ npm install --global markdownlint-cli2
 ```
 
 
+## Oxypng
+
+We use [oxpng](https://github.com/shssoichiro/oxipng) to optimize image file size.
+You can install it with ``cargo install oxipng``
+and then run it with ``oxipng --strip safe --alpha -r src``.
```

Comment:
```suggestion
## Oxipng

We use [oxipng](https://github.com/shssoichiro/oxipng) to optimize image file size.
You can install it with `cargo install oxipng` and then run it as follows:

```bash
oxipng --strip safe --alpha -r src
```
```

Reply:
For some reason the \`\`\` cannot be nested, so the "suggestion" block doesn't render properly.

Either way, I think it's nice if the command has its own line, that makes it a bit easier to see and copy-paste 🙂